duckduckgo/Android
# Remote config (I): create infrastructure

Task/Issue URL: https://app.asana.com/0/1125189844152671/1200676769415755
Tech Design URL: https://app.asana.com/0/1125189844152671/1200721087084600
CC: 

**Description**:
This PR adds the basic infrastructure to be able to use remote config. For now it downloads the privacy config locally and stores some of the data.

**Steps to test this PR**:
1. Install and launch the app
1. In the logcat you should see `Downloading privacy config`
1. Check the database of the privacy config module
1. The table `content_blocking_exceptions` should match the exceptions lists from the contentBlocking section in `https://staticcdn.duckduckgo.com/trackerblocking/config/v1/android-config.json`
1. The table `toggles` should have `contentBlocking` and enabled as 1.


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
